2017-07-25 13:43:53,368 gevent_zmq.py-recv[line:183] ERROR /!\ gevent_zeromq BUG /!\ catching up after missing event (RECV) /!\
2017-07-25 14:21:08,473 gevent_zmq.py-recv[line:183] ERROR /!\ gevent_zeromq BUG /!\ catching up after missing event (RECV) /!\
